src/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,56 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { t } from 'i18next';
+
+import { SafeAreaView, Typography } from '../_components';
+import { useStackNavigation, useToggle } from '../_hooks';
+import { StorageKey } from '../_models';
+import * as Styled from './style';
+
+const Onboarding = () => {
+  const [isChecked, toggleChecked] = useToggle(false);
+  const navigation = useStackNavigation();
+
+  function onPress() {
```

Comment:
functies die als handler dienen voor een button bvb doe ik altijd const. Is dit geen best practise? Had vroeger met binding te maken (wss is dat nu automatisch)